### ISSUE TITLE
Need getting started instructions for this fork

### ISSUE THREAD

**Issue by lzachmann** [csp-inc/landcover#1]
### Getting started

1. start / build the container
```sh
./docker/run.sh
```

2. get the data if you don't have it, activate the conda env, and start the web app
```sh
./setup.sh

3. open your browser to localhost:8080 (clear cache potentially if you are coming back)
```

**Comment by blindjesse** [comment]
Can you give me access to cspinc/ms/landcover/cpu on dockerhub. blindjesse/jesse@csp-inc.org

**Comment by jessjaco** [comment]
Also if you copy the existing files to .mine.json files it gives some warnings. I think the server reads them both. But the .mine.json files are needed. So maybe just touch them?

**Comment by lzachmann** [comment]
Hey Jesse I haven't pushed that image to our registry just yet. Would you like me to, or did you just build locally? Wasn't sure if it was working perfectly just yet -- didn't want to waste my bandwidth on a very large image if not. The touch idea makes total sense. Did you want to go ahead and change the setup shell script?

**Comment by lzachmann** [comment]
To the extent we are coordinating on a specific area / geography, I'm wondering too how you and I can develop a set of *.mine files with shared content? I suppose we could unignore those files, or maybe we just start a new (tiny) repo that includes these shared *.mine files (and possible pointers to data) that we can bring in (with a call to `wget` or `git clone` using our `setup.sh` script)? To start we could try to pull some NAIP imagery for an area like this is CA? https://en.wikipedia.org/wiki/Solar_power_in_California#/media/File:ISS013-E-46315.jpg

**Comment by jessjaco** [comment]
I just ended up not using docker...so no worries. I will edit the setup script when I go back through it.

Here are the digitized solar areas we have, may help use locate a test area(s).
https://code.earthengine.google.com/?asset=projects/GEE_CSP/HM/sources/SolarUtilityScale_DUS2018
https://code.earthengine.google.com/?asset=projects/GEE_CSP/HM/sources/SolarUtilityScale_DW2015

**Comment by lzachmann** [comment]
Yeah no doubt. Seems to be some good test areas, like this one SW of Hatch, NM. The digitized areas appear to be incomplete, meaning these other solar panel arrays went in after that digitization effort was made (c. 2015)? 

![Screen Shot 2020-08-25 at 1 02 40 PM](https://user-images.githubusercontent.com/7574887/91216599-772ac100-e6d3-11ea-963f-89514d29f9ae.png)

Or Lockhart / Kramer Junction?

![Screen Shot 2020-08-25 at 1 07 46 PM](https://user-images.githubusercontent.com/7574887/91217342-9118d380-e6d4-11ea-9c58-d27c7a2da8a4.png)

In terms of the model (I realize this may be putting the cart ahead of the horse), I suppose just detecting change in and of itself would be a sort of win. But having some indication of the _type_ of change (barren --> built, for instance, with whatever probability, and the total land area converted) would be better yet. I can easily envision a dashboard widget that might allow us to select a timeframe (e.g., 2015 - 2020), and then click a button for 'detect change'. We could visualize / map change on a continuous scale as the probability that change has occurred or as a categorical variable -- the actual change type (forested --> field, or barren --> built, for example). Of course, what would be super helpful is an infographic of some sort that shows the magnitude and type of changes happening over that timeframe. All sorts of directions we could go....
